00. Packages to install/1. Genestack python client/README.md

```diff
@@ -24,11 +24,12 @@ python3 -m pip install \
       genestack-client==1.53.12
 ```
 
-2. Obtain a token for the Public user in Genestack by logging in to ODM as the Public user and clicking on the profile link under the Public username:
+2. Obtain a token for your Genestack user by logging in to ODM and clicking on the profile link under the username. 
```

Comment:
Let's add info that the user has to click on the button "Create new token" to receive the email